### ISSUE TITLE
Fix data.json (self-test revealed violations)

### ISSUE THREAD

**Issue by torerobo** [sherlock-project/sherlock#487]
Update:
Correction of a self-test error.
====

Bad
```
"Insanejournal": {
    "errorMsg": "Unknown user",
    "errorType": "message",
    "rank": 28570,
    "url": "http://{}.insanejournal.com/profile",
    "urlMain": "insanejournal.com",
    "username_claimed": "blue",
    "username_unclaimed": "noneownsthisusername" 
```
Fix
```
"Insanejournal": {
    "errorMsg": "Unknown user",
    "errorType": "message",
    "rank": 28570,
    "url": "http://{}.insanejournal.com/profile",
    "urlMain": "insanejournal.com",
    "username_claimed": "blue",
    "username_unclaimed": "dlyr6cd" 
```

====

"**KiwiFarms**" Website - Permanent False positives
On a nonexistent profile, it displays many different messages in the browser.

====

Bad
```
  "Brew": {
    "errorType": "status_code",
    "rank": 377614,
    "url": "https://www.brew.com/{}",
    "urlMain": "https://www.brew.com/",
    "urlProbe": "https://api.brew.com/api/public/podcast/{}",
    "username_claimed": "blue",
    "username_unclaimed": "noonewouldeverusethis7"
  },

```
Fix
```
  "Brew": {
    "errorType": "status_code",
    "rank": 377614,
    "url": "https://www.brew.com/{}",
    "urlMain": "https://www.brew.com/",
    "username_claimed": "blue",
    "username_unclaimed": "noonewouldeverusethis7"
  },
```

====

 **"Gitee"**
```
  "Gitee": {
    "errorType": "status_code",
    "rank": 7151,
    "url": "https://gitee.com/{}",
    "urlMain": "https://gitee.com/",
    "username_claimed": "blue",
    "username_unclaimed": "noonewouldeverusethis7"
  },
```
Remove site. Nickname "blue" and the like in Latin do not exist.

====

"LOR"
Bad
```
"LOR": {
    "errorType": "status_code",
    "rank": 26475,
    "url": "https://www.linux.org.ru/people/{}/profile",
    "urlMain": "https://linux.org.ru/",
    "username_claimed": "blue", 
    "username_unclaimed": "noonewouldeverusethis7"
   },
```
Fix
```
"LOR": {
    "errorType": "status_code",
    "rank": 26475,
    "url": "https://www.linux.org.ru/people/{}/profile",
    "urlMain": "https://linux.org.ru/",
    "username_claimed": "red",
    "username_unclaimed": "noonewouldeverusethis7"
   },
```
====

http://fr.viadeo.com/en - Is the site broken? (not available with usa / ru ip addresses).

====
Bad
```
 "Zomato": {
    "errorType": "status_code",
    "headers": {
      "Accept-Language": "en-US,en;q=0.9"
    },
    "rank": 859,
    "url": "https://www.zomato.com/{}/foodjourney",
    "urlMain": "https://www.zomato.com/",
    "username_claimed": "deepigoyal",
    "username_unclaimed": "noonewouldeverusethis7"
  },
```
Fix
```
 "Zomato": {
    "errorType": "status_code",
    "headers": {
      "Accept-Language": "en-US,en;q=0.9"
    },
    "rank": 859,
    "url": "https://www.zomato.com/pl/{}/foodjourney",
    "urlMain": "https://www.zomato.com/",
    "username_claimed": "deepigoyal",
    "username_unclaimed": "noonewouldeverusethis7"
   },
```

====

Bad
```
  "toster": {
    "errorType": "status_code",
    "rank": 7718,
    "url": "https://www.toster.ru/user/{}/answers",
    "urlMain": "https://www.toster.ru/",
    "username_claimed": "blue",
    "username_unclaimed": "noonewouldeverusethis7"
  },
```
Fix
```
  "toster": {
    "errorType": "status_code",
    "rank": 7718,
    "url": "https://www.toster.ru/user/{}/answers",
    "urlMain": "https://www.toster.ru/",
    "username_claimed": "adam",
    "username_unclaimed": "noonewouldeverusethis7"
  },
```

**Comment by hoadlck** [comment]
"Gitee" does seem to work.  I found a username.

**Comment by hoadlck** [comment]
I am able to access "Viadeo" from USA.  The tests do work for that site.

**Comment by hoadlck** [comment]
Included changes in #499.  Except for "Gitee" and "Viadeo" as mentioned above.
